fastlane/Fastfile

```diff
@@ -56,9 +56,6 @@ SUPPORTED_LOCALES = [
 # Environment
 ########################################################################
 Dotenv.load(USER_ENV_FILE_PATH)
-ENV["PROJECT_NAME"] = "pocket-casts-android"
-ENV["PROJECT_ROOT_FOLDER"] = "./"
-ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'main'
```

Comment:
ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] was used by some of the versioning actions but isn't needed anymore about the [recent versioning updates](https://github.com/Automattic/pocket-casts-android/pull/1425).